messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsConnector.java

```diff
@@ -392,6 +397,37 @@ public SubscriberBuilder<? extends Message<?>, Void> getSubscriberBuilder(Config
         }
     }
 
+    private void configureProducer(MessageProducer producer, ConnectionContext ctx) {
+        io.helidon.config.Config config = ctx.config().get("producer");
+        if (!config.exists()) return;
+
+        Class<? extends MessageProducer> clazz = producer.getClass();
+        Map<String, Method> setterMethods = Arrays.stream(clazz.getDeclaredMethods())
```

Comment:
This is using reflection - if the clazz comes from Helidon, we should add it for reflection in this module. Otherwise we should add all implementing classes of `MessageProducer` to helidon native image configuration